packages/core/src/Synchronizer.ts

```diff
@@ -316,6 +321,41 @@ export class Synchronizer extends EventEmitter {
         })
     }
 
+    get blocks() {
```

Comment:
I don't think we need getters here, these variables should not be accessed outside of this class

Reply:
removed